contact.html

```diff
@@ -0,0 +1,119 @@
+              <div class="icon_line_1 _2vsXM"></div>
+              <div class="icon_line_1 icon_line_1_2 icon_line_1_3"></div>
+            </div>
+            <div class="nav_container" >
+                <nav>
+                  <ul class="font2" style="font-size: 13px;">
+                    <li>
+                      <a href="/index.html">ABOUT ME</a>
+                      </li>
+                        <li>
+                          <a href="#">RESUME</a>
+                        </li>
+                        <li>
+                          <a href="#">PROJECTS</a>
+                        </li>
+                        <li>
+                          <a href="/contact.html">CONTACT</a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+    </div>
+    </header>
+
+    <main id="contact_container" style="background-image:var(--grey1);background-color: var(--grey1);border-bottom: dotted; border-bottom-color: transparent;">
+        <section class="contact_header">
+            <div class="square_icon" id="square_2">
+                <div id="align" class="square_1" style="margin: inherit;">
+                  <div id="square_2inlineContent" data-testid="inline-content" class="">
+                    <div
+                      id="square_2inlineContent-gridContainer"
+                      data-testid="mesh-container-content"
+                    ></div>
+                  </div>
+                </div>
+              </div>
+
+            <div id="bigger" class="contact_slogan font1">Let's talk</div>
+        </section>
+
+        <section class="contact_section font2" style="font-weight:bold;">
+            <form>
+                <label id="first_name"> First Name *</label> <br>
+                <input type="text" name="first_name" id="first_name" required /> <br>
+                <label id="last_name"> Last Name *</label> <br>
+                <input type="text" name="last_name" id="last_name" required /> <br>
+                <label id="email"> Email *</label> <br>
+                <input type="text" name="email" id="email" required /> <br>
+                <label id="subject"> Subject</label> <br>
+                <input type="text" name="subject" id="subject" /> <br>
+                <label id="message">Message</label> <br>
+                <textarea></textarea><br>
+                <div style="text-align: center;"><button class="button1" style="width: 100px;" type="submit"> SEND</button></div>
+            </form>
+        </section>
+    </main>
+    <footer>
+        <div class="tel">
+          <p class="footer_text">Call</p>
+          <p>123-456-7890</p>
+        </div>
+        <div class="mail">
+          <p class="footer_text">Write</p>
+          <p>info@mysite.com</p>
+        </div>
+        <div class="media">
+          <p class="footer_text">Follow</p>
+          <div>
+              <img src="/designs/fb.png"  class="icons"></img>
```

Comment:
remember to use an `alt` tag for images, this is so people that rely on assistive technology like screen readers can still understand what the image is